client/app/services/HelpTrigger.jsx

```diff
@@ -20,16 +31,121 @@ export default class HelpTrigger extends React.PureComponent {
     type: PropTypes.oneOf(Object.keys(TYPES)).isRequired,
   }
 
+  iframeRef = null
+
+  iframeLoadingTimeout = null;
+
+  constructor(props) {
+    super(props);
+    this.iframeRef = React.createRef();
+  }
+
+  state = {
+    visible: false,
+    loading: false,
+    error: false,
+  }
+
+  componentWillUnmount() {
+    this.onDrawerClosed();
```

Comment:
This method is supposed to exist? :thinking:

Reply:
lol used to. Fixed now. TS WE AWAIT THEE!!